schema/3pse/file/applications/espresso/7.2/pw.x/system.json

```diff
@@ -0,0 +1,638 @@
+            "type": "string",
+            "description": "Exchange-correlation functional: eg 'PBE', 'BLYP' etc See Modules/funct.f90 for allowed values. Overrides the value read from pseudopotential files. Use with care and if you know what you are doing!"
+        },
+        "ace": {
+            "type": "boolean",
+            "default": true,
+            "description": "Use Adaptively Compressed Exchange operator as in Lin Lin, J. Chem. Theory Comput. 2016, 12, 2242--2249, doi:10.1021/acs.jctc.6b00092"
+        },
+        "exx_fraction": {
+            "type": "number",
+            "description": "Fraction of EXX for hybrid functional calculations. In the case of input_dft='PBE0', the default value is 0.25, while for input_dft='B3LYP' the exx_fraction default value is 0.20."
+        },
+        "screening_parameter": {
+            "type": "number",
+            "default": 0.106,
+            "description": "screening_parameter for HSE like hybrid functionals."
+        },
+        "exxdiv_treatment": {
+            "type": "string",
+            "default": "gygi-baldereschi",
+            "enum": [
+                "gygi-baldereschi",
+                "vcut_spherical",
+                "vcut_ws",
+                "none"
+            ]
+        },
+        "x_gamma_extrapolation": {
+            "type": "boolean",
+            "default": true,
+            "description": "Specific for EXX. If .true., extrapolate the G=0 term of the potential"
+        },
+        "ecutvcut": {
+            "type": "number",
+            "description": "Reciprocal space cutoff for correcting Coulomb potential divergencies at small q vectors."
+        },
+        "nqx1": {
+            "type": "integer",
+            "default": 0,
+            "description": "Three-dimensional mesh for q (k1-k2) sampling of the Fock operator (EXX). Can be smaller than the number of k-points."
+        },
+        "nqx2": {
+            "type": "integer",
+            "description": "Three-dimensional mesh for q (k1-k2) sampling of the Fock operator (EXX). Can be smaller than the number of k-points."
+        },
+        "nqx3": {
+            "type": "integer",
+            "description": "Three-dimensional mesh for q (k1-k2) sampling of the Fock operator (EXX). Can be smaller than the number of k-points."
+        },
+        "localization_thr": {
+            "type": "number",
+            "default": 0,
+            "description": "Overlap threshold over which the exchange integral over a pair of localized orbitals is included in the evaluation of EXX operator. Any value greater than 0.0 triggers the SCDM localization and the evaluation on EXX using the localized orbitals. Very small value of the threshold should yield the same result as the default EXX evaluation"
+        },
+        "Hubbard_occ": {
+            "type": "array",
+            "items": {
+                "type": "array",
+                "items": {
+                    "type": "number"
```

Comment:
Hi @k0stik, just a suggestion, here we could enforce more stricter tuple validation:
```json
"Hubbard_occ": {
    "type": "array",
    "items": {
        "type": "array",
        "items": [
            { "type": "integer", "minimum": 1 },
            { "type": "integer", "minimum": 1, "maximum": 3 },
            { "type": "number" }
        ],
        "maxItems": 3,
        "minItems": 3
    }
},
```
First two items in the array are indices of atomic species and Hubbard manifold. Both are non-negative integers and there could be up to 3 Hubbard manifolds. Note that if you accept this suggestion, we need to update the example data:
```
"Hubbard_occ": [
            [1, 1, 0.0],
            [1, 2, 0.02]
        ]
```